app/models/organisation.rb

```diff
@@ -8,9 +8,9 @@ class Organisation < ActiveRecord::Base
 
   has_many :permissions
   has_many :memberships
-  has_many :profiles, through: :memberships
+  has_many :profiles, -> { order :name }, through: :memberships
```

Comment:
Not very sure about this. How do you feel about moving this to the default scoping on the `Profile` model?